Applications/FileManager/FileUtils.cpp

```diff
@@ -35,6 +35,16 @@
 
 namespace FileUtils {
 
+int delete_file(String path)
+{
+
+    if (unlink(path.characters()) < 0) {
+        return errno;
+    }
+    dbg() << "FileUtils: unlink(%s" << path << ") succeeded";
```

Comment:
`%s`?

Reply:
I must have missed that when converting from the dbgprintf() to dbg().

Reply:
I must have missed that when converting from the dbgprintf() to dbg().